app/scripts/modules/core/src/cluster/rollups.less

```diff
@@ -275,6 +275,7 @@ running-tasks-tag {
     display: block;
   }
   a {
+    cursor: pointer;
```

Comment:
You don't get a pointer unless you have an `href`. You guys probably don't know this, but `a` stands for ⚓️ , and ⚓️ s can be used to navigate within a page. It's pretty complex, not sure if you would understand.